src/pages/Main/Main.js

```diff
@@ -0,0 +1,178 @@
+        <div className="upperMargin">
+          <div className="mainBestList">
+            <div className="bestListTitleContainer">
+              <div className="bestListTitle">BEST</div>
+              <a href="#home" className="bestListLink">
+                + SHOP
+              </a>
+            </div>
+            <ul className="bestListProductRecommend">
+              <li className="bestListProduct">
+                <div>컴포넌트</div>
+              </li>
+              <li className="bestListProduct">
+                <div>컴포넌트</div>
+              </li>
+              <li className="bestListProduct">
+                <div>컴포넌트</div>
+              </li>
+            </ul>
+          </div>
+          <div className="mainBestList">
+            <div className="bestListTitleContainer">
+              <div className="bestListTitle">NEW</div>
+              <a href="#home" className="bestListLink">
+                + SHOP
+              </a>
+            </div>
+            <ul className="bestListProductRecommend">
+              <li className="bestListProduct">
+                <div>컴포넌트</div>
+              </li>
+              <li className="bestListProduct">
+                <div>컴포넌트</div>
+              </li>
+              <li className="bestListProduct">
+                <div>컴포넌트</div>
+              </li>
+              <li className="bestListProduct">
+                <div>컴포넌트</div>
+              </li>
+            </ul>
+          </div>
+          <div className="mainBannerContainer">
+            <img src="./images/coffee1.jpg" alt="coffee" />
+            <ul className="mainBannerSection">
+              {BannerCardData.map(data => {
+                return (
+                  <li className="mainBannerCard" key={data.id}>
+                    <div className="cardTitle">{data.Title}</div>
+                    <div className="cardSubTitle">{data.subTitle}</div>
+                  </li>
+                );
+              })}
+            </ul>
+          </div>
+          <div className="mainBottomLink">
+            <ul className="mainBottomContainer">
+              <li>
+                <a href="#home">
+                  <img
+                    src="https://terarosa.com/SkinImg/main_bt_01.jpg"
+                    alt="icon1"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#home">
+                  <img
+                    src="https://terarosa.com/SkinImg/main_bt_02.jpg"
+                    alt="icon2"
+                  />
+                </a>
+              </li>
+              <li>
+                <a href="#home">
+                  <img
+                    src="https://terarosa.com/SkinImg/main_bt_03.jpg"
+                    alt="icon3"
+                  />
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+}
+
+export default Main;
+
+const ImageData = [
```

Comment:
```suggestion
const IMAGE_DATA = [
```
상수데이터 이름은 전체 대문자랑 snake_case로 구현해야 합니다

Reply:
네 알겠습니다